### ISSUE TITLE
replace LightWeightPolyline with LwPolyline

### ISSUE THREAD

**Issue by Yahasana** [haplokuon/netDxf#192]
it's not a bug but please replace all `LightWeightPolyline` with `LwPolyline`. 
e.g. `DxfObjectCode.LightWeightPolyline` an `SubclassMarker.LightWeightPolyline`

**Comment by haplokuon** [comment]
Yeah, I guess I should have done that the day I renamed the LwPolyline class long time ago, since that is the name given in the DXF, although it means the same thing. I guess that, since those constants are mainly for internal use of the library I forgot to change them to abbreviate.

**Comment by haplokuon** [comment]
It is just a simple rename, it should have no side effects as long as it only affects the DxfObjectCode and the SubclassMarker constants.
